### PR TITLE
test

### PR DESCRIPTION
**Pull request by anhdh-3069**

### CODE REVIEW

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

File: README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+ddđ
```
Comment (manhnt-0960) [inline README.md:25]: cmt

**Review by manhnt-0960 — COMMENTED** [review]
reject

**Comment by anhdh-3069** [comment]
ready

**Comment by manhnt-0960** [comment]
ready

**Comment by manhnt-0960** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

File: README.md [hunk 2]
```diff
@@ -22,3 +22,4 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+ddđđáada
```
Comment (manhnt-0960) [inline README.md:25]: cmt

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

File: README.md (hunk 2 above)
Comment (manhnt-0960) [inline README.md:25]: cmt

**Comment by anhdh-3069** [comment]
ready

File: README.md (hunk 2 above)
Comment (manhnt-0960) [inline README.md:25]: cmt

**Comment by anhdh-3069** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request closed without merging**